modules/@apostrophecms/i18n/index.js

```diff
@@ -1251,6 +1257,53 @@ module.exports = {
             console.log(`Due to conflicts, kept ${kept} documents from ${keep}`);
           }
         }
+      },
+      'strip-slug-accents': {
+        usage: 'Remove Latin accent characters from all document slugs and attachment names. Usage: node app @apostrophecms/i18n:strip-slug-accents',
+        async task() {
+          let docChanged = 0;
+          let attachmentChanged = 0;
+
+          await self.apos.migration.eachDoc({}, 5, async doc => {
+            const slug = doc.slug;
+            const req = self.apos.task.getAdminReq({
+              locale: doc.aposLocale?.split(':')[0] || 'en'
```

Comment:
Forcing the `locale` to `en` seems wrong. We have `i18n` `defaultLocale` for that.
If you're using `doc.aposLocale` you can also pass it as `query: { aposLocale: doc.aposLocale }` and let the i18n middleware do its magic.